### PR TITLE
fix(Users/Quota setting): Prevent floating point value from getting truncated in locales other than english

### LINKED ISSUE #18468: Cannot set user quota with uneven values using locales that use a comma as decimal separator
It is not possible to set a decimal user quota with a German locale. The frontend does change the wanted value of e.g "1.5 TB" in the API request to "1,5 TB" which is not correctly interpreted by the backend but still accepted and returns a 200 OK message.

If you try to set "1,5 TB" then is the API request not even sent.

There is no problem if the English locale is activated for the user and it is tried to set "1.5 TB".

### Steps to reproduce
1. Go to User Overview
2. Try to set a Quota of "1.5 TB" for a user

### Expected behaviour
User gets a quota of 1.5 TB quota.

### Actual behaviour
Frontend shows "Unlimited" quota and after reload of the page "1 TB".

### Server configuration

**Operating system:** Ubuntu Server 18.04

**Web server:** Nginx

**Database:** MariaDB

**PHP version:** PHP 7.3.11

**Nextcloud version:** 17.0.1

**Updated from an older Nextcloud/ownCloud or fresh install:** upgraded from Nextcloud version 16

**Where did you install Nextcloud from:** Public Nextcloud Docker Image

**Signing status:**
<details>
<summary>Signing status</summary>
No errors have been found.
</details>

**Are you using external storage, if yes which one:** No

**Are you using encryption:** No

**Are you using an external user-backend, if yes which one:** No

### Client configuration
**Browser:** Chromium and Firefox

**Operating system:** Fedora

#### Web server error log
<details>
<summary>Web server error log</summary>
No errors
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
No errors
</details>

#### Browser log
<details>
<summary>Browser log</summary>

##### Client Request:
Path: `example.org/ocs/v2.php/cloud/users/Admin`
Payload: `{key: "quota", value: "1,5 TB"}`

##### Answer:
```json
{"ocs":{"meta":{"status":"ok","statuscode":200,"message":"OK"},"data":[]}}
```
</details>

### PR DESCRIPTION
**Pull request by marcelklehr**
* Resolves: #18468

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

File: apps/settings/src/components/Users/UserRow.vue [hunk 1]
```diff
@@ -307,6 +307,7 @@ import UserRowActions from './UserRowActions.vue'
 
 import UserRowMixin from '../../mixins/UserRowMixin.js'
 import { isObfuscated, unlimitedQuota } from '../../utils/userUtils.ts'
+import {formatFileSize, parseFileSize} from "@nextcloud/files";
```
Comment (skjnldsv) [inline apps/settings/src/components/Users/UserRow.vue:310]:
```suggestion
import {formatFileSize, parseFileSize} from '@nextcloud/files'
```

**Comment by marcelklehr** [comment]
/backport https://github.com/nextcloud/server/commit/162f8017448d19efe9c4278ee89736e7b38f8d3b to stable27

**Pull request merged**